tests/scripts/power-macro

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+mkdir "$PROG_HOME/tests/power-macro/out"
+
+dotc -d "$PROG_HOME/tests/power-macro/out" "$PROG_HOME/tests/PowerMacro.scala"
```

Comment:
`dotc` is not available. You can change to `compile`, which I'm going to provide.